phone/src/locale/ar.json

```diff
@@ -0,0 +1,315 @@
+      },
+      "SEARCH_PLACEHOLDER": "رسائل البحث...",
+      "DELETE_CONVERSATION": "مسح المحادثة",
+      "NEW_MESSAGE": "رسالة...",
+      "NEW_MESSAGE_GROUP": "أرقام الهواتف...",
+      "SHARE_IMAGE_TITLE": "هل تريد مشاركة هذه الصورة؟",
+      "SHARE_CONTACT_TITLE": "هل تريد مشاركة جهة الاتصال هذه؟",
+      "GROUP_CHAT_LABEL": "تصنيف الدردشة الجماعية ...",
+      "MEDIA_OPTION": "الكاميرا / المعرض",
+      "CONTACT_OPTION": "شارك جهة الإتصال",
+      "CONTACT_SHARED": "تمت مشاركة جهة الاتصال",
+      "LOCATION_OPTION": "مشاركة الموقع",
+      "LOCATION_MESSAGE": "تمت مشاركة الموقع",
+      "LOCATION_TOOLTIP": "تعيين إحداثية",
+      "NEW_MESSAGE_GROUP_SUBMIT": "إرسال",
+      "INPUT_NAME_OR_NUMBER": "ابحث عن جهات الاتصال أو اكتب الرقم واضغط على Enter",
+      "ACTIONS_TITLE": "أجراءات",
+      "ME": "أنا"
+    },
+    "RELATIVE_TIME": {
+      "future": "بعد %s",
+      "past": "قبل %s",
+      "s": "بضع ثوان",
+      "m": "دقيقة",
+      "mm": "%d دقائق",
+      "h": "ساعة",
+      "hh": "%d ساعات",
+      "d": "يوم",
+      "dd": "%d ايام",
+      "M": "شهر",
+      "MM": "%d اشهر",
+      "y": "سنة",
+      "yy": "%d سنوات"
+    },
+    "GENERIC": {
+      "LOADING": "جار التحميل...",
+      "WAIT": "أرجو الإنتظار...",
+      "SAVE": "حفظ",
+      "EDIT": "تعديل",
+      "UPDATE": "تحديث",
+      "REQUIRED": "مطلوب",
+      "DELETE": "حذف",
+      "REPORT": "أبلغ عن",
+      "CALL": "مكالمة",
+      "MESSAGE": "رسالة",
+      "CLOSE": "أغلق",
+      "CANCEL": "إلغاء",
+      "TITLE": "عنوان",
+      "CONTENT": "محتوى",
+      "SHARE": "شارك",
+      "ADD": "أضف",
+      "WRITE_TO_CLIPBOARD_TOOLTIP": "نسخ {{ content }}",
+      "WRITE_TO_CLIPBOARD_MESSAGE": "تم نسخه {{content}} إلى الحافظة!"
+    },
+    "COMING_SOON": "قريبا...",
+    "INITIALIZING": "جار تهيئة",
+    "APPS_DIALER": "هاتف",
+    "APPS_CONTACTS": "جهات الاتصال",
+    "APPS_MESSAGES": "رسائل",
+    "APPS_CALCULATOR": "آلة حاسبة",
+    "APPS_SETTINGS": "إعدادات",
+    "APPS_BANK": "WhoDis Banking",
+    "APPS_NOTES": "ملاحظات",
+    "APPS_EXAMPLE": "Example app",
+    "APPS_TWITTER": "تويتر",
+    "APPS_MARKETPLACE": "المتجر",
+    "APPS_CAMERA": "كاميرا",
+    "APPS_MATCH": "وسيط زواج",
+    "APPS_CALCULATOR_COPIED": "تم نسخ النتيجة إلى الحافظة!",
+    "GENERIC_CLICK_TO_REVEAL": "انقر للكشف",
+    "GENERIC_LOADING": "جار التحميل...",
+    "GENERIC_WAIT": "أرجو الإنتظار...",
+    "GENERIC_SAVE": "حفظ",
+    "GENERIC_EDIT": "تعديل",
+    "GENERIC_UPDATE": "تحديث",
+    "GENERIC_REQUIRED": "مطلوب",
+    "GENERIC_DELETE": "حذف",
+    "GENERIC_REPORT": "أبلغ عن",
+    "GENERIC_CALL": "مكالمة",
+    "GENERIC_MESSAGE": "رسالة",
+    "GENERIC_CLOSE": "أغلق",
+    "GENERIC_CANCEL": "إلغاء",
+    "GENERIC_TITLE": "عنوان",
+    "GENERIC_CONTENT": "محتوى",
+    "GENERIC_SHARE": "شارك",
+    "GENERIC_ADD": "أضف",
+    "GENERIC_WRITE_TO_CLIPBOARD_TOOLTIP": "نسخ {{ content }}",
+    "GENERIC_WRITE_TO_CLIPBOARD_MESSAGE": "تم نسخه {{content}} إلى الحافظة!"
```

Comment:
There should be a comma at the end of this line for the json file to be valid

Reply:
Yes, i fixed the mistake while testing, guess I copied the wrong file